### ISSUE TITLE
/junta command

### ISSUE THREAD

**Issue by Ironjanowar** [acmfi/acm_bot#7]
**Is your feature request related to a problem? Please describe.**
Someone may need to talk to someone of the ACM board and he/she will have to ask in the general chat and dodge some trolls.

**Describe the solution you'd like**
It would be useful to have a `/junta` command that shows the aliases (and the roles) of ACM UPM board, in case someone wants to ask something.

**Describe alternatives you've considered**
None.

**Additional context**
Thanks to @AshidaSRS for the idea..

**Comment by Ironjanowar** [comment]
To achieve this I thought of a json file with all the info, something like:
```json
[
  {
    "username": "@SamRadioFloyd",
    "role": "President (like the cheese)"
  },
  {
    "username": "@DefaultValue",
    "role": "Treasurer (not like the cheese)"
  }
]
```

But is this enough information to have at least an SQLite?

**Comment by Alfedi** [comment]
Maybe a  json file with basic information (such as name, Telegram alias and roles) is enough. I don't think SQLite is will be necesary because there isn't so much information to store in.

**Comment by rockneurotiko** [comment]
Agree with @Alfedi let's just use a JSON/YAML here

**Comment by Ironjanowar** [comment]
Solved in #11
